src/TropicalGeometry/TropicalPolyhedron/constructors.jl

```diff
@@ -0,0 +1,151 @@
+  pm_tpolytope::Polymake.BigObject
+
+  @doc raw"""
+      TropicalPointConfiguration{M}(p::Polymake.BigObject) where M<:MinOrMax
+
+  Constructs a `TropicalPointConfiguration` corresponding to a `Polymake.BigObject` with tropical addition `M`.
+  """
+  TropicalPointConfiguration{M}(p::Polymake.BigObject) where {M<:MinOrMax} = new{M}(p)
+end
+
+pm_object(P::TropicalPolyhedron) = P.pm_tpolytope
+pm_object(P::TropicalPointConfiguration) = P.pm_tpolytope
+
+convention(::TropicalPolyhedron{typeof(min)}) = min
+convention(::TropicalPolyhedron{typeof(max)}) = max
+convention(::TropicalPointConfiguration{typeof(min)}) = min
+convention(::TropicalPointConfiguration{typeof(max)}) = max
+
+tropical_convex_hull(P::TropicalPointConfiguration{M}) where {M<:MinOrMax} = TropicalPolyhedron{M}(pm_object(P))
+tropical_point_configuration(P::TropicalPolyhedron{M}) where {M<:MinOrMax} = TropicalPointConfiguration{M}(pm_object(P))
+
+tropical_convex_hull(V::AbstractVector{T}) where {T<:AbstractVector{<:TropicalSemiringElem}} = tropical_convex_hull(matrix(V))
+tropical_point_configuration(V::AbstractVector{T}) where {T<:AbstractVector{<:TropicalSemiringElem}} = tropical_point_configuration(matrix(V))
+
+@doc raw"""
+    tropical_convex_hull(V::MatElem{TropicalSemiringElem{M}}) where M<:MinOrMax
+
+Constructs the tropical convex hull of the vertices `V` in the tropical projective torus
+with `M` as tropical addition.
+
+# Examples
+Defines a tropical unit ball in the 2-dimensional tropical projective torus with `max` as addition:
+```jldoctest tropical_convex_hull
+julia> TT = tropical_semiring(max);
+
+julia> A = TT.(identity_matrix(QQ, 3));
+
+julia> P = tropical_convex_hull(A)
+Max tropical polyhedron in tropical projective torus of dimension 2
+```
+
+The vertices of a tropical convex hull may also be given by the rows of a tropical matrix:
+```jldoctest tropical_convex_hull
+julia> A = TT[0 -4 -1; 0 -1 0];
+
+julia> P = tropical_convex_hull(A)
+Max tropical polyhedron in tropical projective torus of dimension 2
+```
+"""
+tropical_convex_hull(V::MatElem{TropicalSemiringElem{M}}) where {M<:MinOrMax} = _tropical_convex_hull(TropicalPolyhedron, V)
+@doc raw"""
+    tropical_point_configuration(V::MatElem{TropicalSemiringElem{M}}) where M<:MinOrMax
+
+Constructs the tropical point configuration of the vertices `V` in the tropical projective torus
+with `M` as tropical addition.
+
+# Examples
+Defines a point configuration containing a tropical unit ball in the 2-dimensional tropical projective torus with `max` as addition:
+```jldoctest tropical_point_configuration
+julia> TT = tropical_semiring(max);
+
+julia> A = TT.(identity_matrix(QQ, 3));
+
+julia> P = tropical_point_configuration(A)
+Max tropical point configuration in tropical projective torus of dimension 2
+```
+The tropical point configuration may also be given by the rows of a tropical matrix:
+```jldoctest tropical_point_configuration
+julia> A = TT[0 -4 -1; 0 -1 0];
```

Comment:
```suggestion
julia> A = T[0 -4 -1; 0 -1 0];
```

Consistency with other tropical docu.